x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -265,6 +266,34 @@ export function ServiceInventory() {
     [tiebreakerField]
   );
 
+  const setApplicationContext =
+    useApmPluginContext().observabilityAIAssistant.service
+      .setApplicationContext;
+
+  useEffect(() => {
+    if (isFailure(mainStatisticsStatus)) {
+      return setApplicationContext({
+        description: 'The services have failed to load',
+      });
+    }
+
+    if (isPending(mainStatisticsStatus)) {
+      return setApplicationContext({
+        description: 'The services are still loading',
```

Comment:
What happens if the user initiates the conversation before the context has loaded?

Reply:
Then this description is passed to the LLM, or do you mean something else?

Reply:
When I wrote this I thought we were adding the consumer provided context to the system prompt. I now see that it is part of the `context` function so is sent to the LLM whenever it requests it (which might be never, no?)

Reply:
... btw do we send over "The services are still loading" to the LLM? If so, why?